### ISSUE TITLE
Validation errors causes inc/dec button decoration to displace.

### ISSUE THREAD

**Issue by Abhilash-Chandran** [Abhilash-Chandran/number_inc_dec#29]
Whenever a validation error is shown, the inc/dec buttons borders are displaced. 

![image](https://user-images.githubusercontent.com/4489849/92129293-9cba7780-ee03-11ea-9a37-8b6bb8a7c3c8.png)

This is partially due to the way the Ink widget works and cannot be fixed in its current form. An Ink widget is required to give the ripple effect. But the border issue is cause t due to Ink widget not applying the decoration to its child. Check this.

https://github.com/flutter/flutter/issues/21074

So it can be replaced with the solution proposed in the following issue until the above issue is fixed.
https://github.com/flutter/flutter/issues/3782

Basically replace Ink with a Material widget and type as transparency and wrap it in a container to enforce the decoration.
